travispy/entities/_entity.py

```diff
@@ -192,8 +192,7 @@ def _load(cls, infos, session):
                 try:
                     setattr(entity, key, value)
                 except AttributeError:
-                    warn('Unknown {0} attribute {1}'
-                         .format(entity.__class__.__name__, key))
+                    pass
```

Comment:
I really don't like the idea of ignoring this. I do prefer sending it to `log.warning` giving the option to set the log level.

Reply:
That's because the import from log causes a circular import... entity imports log, log imports import. The log here isn't a python logger, it's a special log class. It also doesn't have warn or debug on it.